src/Generation/GapicClientGenerator.php

```diff
@@ -443,7 +443,7 @@ private function getClientDefaults(): PhpClassMember
 
         // TODO: Consolidate setting all the known array values together.
         // We do this here to maintain the existing sensible ordering.
-        if ($this->serviceDetails->transportType !== Transport::REST) {
+        if ($this->serviceDetails->transportType === Transport::GRPC_REST) {
```

Comment:
reverting this change simply because `GapicClientGenerator` is considered "deprecated". But this isn't harming anyone how it was.